server/item/enchantment/vanishing.go

```diff
@@ -35,15 +35,12 @@ func (Vanishing) CompatibleWithEnchantment(t item.EnchantmentType) bool {
 
 // CompatibleWithItem ...
 func (Vanishing) CompatibleWithItem(i world.Item) bool {
-	switch i.(type) {
-	case item.Durable:
-		//case block.Skull:
-		//case block.Pumpkin:
-		//case block.LitPumpkin:
-		// note: causes import cycle
-		return true
-	}
-	return false
+	_, dur := i.(item.Durable)
+	_, com := i.(item.Compass)
```

Comment:
Include `item.Armour` for skulls/carved pumpkins

Reply:
sorry, was thinking like "armor is durable" then realized pumpkins/skulls are not actually durable attachables 😅